apps/camera/js/lib/camera/camera.js

```diff
@@ -889,6 +905,9 @@ Camera.prototype.startRecording = function(options) {
       }
 
       video.filepath = filepath;
+      video.poster.filepath = filepath.replace('.3gp', '.jpg');
```

Comment:
I think that the DCIF standard says that these should be .thm files or something like that instead of .jpg. If we did that, we'd have to change the device storage config to recognize .thm files, and we'd have to change gallery to recognize them too.  Not something we need to do now, but something I regret not doing from the start.